quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/actions/MovePawn.java

```diff
@@ -0,0 +1,15 @@
+package com.rl.mpquoridor.models.actions;
+
+import com.rl.mpquoridor.models.enums.MovementDirection;
+
+public class MovePawn extends  TurnAction {
```

Comment:
call it MovePawnAction - class must be a noun and methods must be verbs
MovePawn is a verb to make it noun use Action in the end